src/rpn/rpn.test.ts

```diff
@@ -1,21 +1,33 @@
-import { rpn } from "./rpn";
+import {rpn} from "./rpn";
 
 describe("rpn()", () => {
-    it("returs number when passed", () => {
-      expect(rpn("2")).toBe(2);
+    it("returns number when passed", () => {
+        expect(rpn("2")).toBe(2);
     });
 
+    it("correctly adds 2 numbers, when 0 included", () => {
+        expect(rpn("2 0 +")).toBe(2);
+    });
 
-  it("correctly adds 2 numbers", function () {
-    expect(rpn("2 0 +")).toBe(2);
-  });
+    it(`correctly adds 2 numbers`, () => {
+        expect(rpn("2 2 +")).toBe(4);
+    });
 
+    it("correctly divides  2  numbers", () => {
+        expect(rpn("2 2 /")).toBe(1);
+    });
 
-  it(`correctly adds 2 numbers`, async () => {
-    expect(rpn("2 2 +")).toBe(4);
-  });
+    it(`more than one operation and multiplication is performed correctly `, () => {
+        expect(rpn("3 1 – 2 2 + *")).toBe(8);
+    });
 
-  it("correctly divides  2  numbers", () => {
```

Comment:
Instead of this test case, I added 3 test cases handling the error thrown when validating the input data.